libs/wingsdk/src/std/array.ts

```diff
@@ -249,4 +249,19 @@ export class MutArray {
   public pop(): T1 {
     throw new Error("Abstract");
   }
+
+  /**
+   * Sets a new value at the given index of an array
+   *
+   * @macro ((obj, args) => { if (args[0] < 0 || args[0] >= $self$.length) throw new Error("Index out of bounds"); obj[args[0]] = args[1]; })($self$, [$args$])
```

Comment:
```suggestion
   * @macro ((obj, args) => { if (args[0] < 0 || args[0] >= ($self$.length -1)) throw new Error("Index out of bounds"); obj[args[0]] = args[1]; })($self$, [$args$])
```

Reply:
@tsuf239  Since it is evaluating `greater than or equal`, isn't `$self$.length` what is needed? Since the max index can only be `$self$.length -1`. Or does `$self$.length -1` do something different than what I think it does?

Reply:
No, you're actually right, resolving this comment